WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -21,25 +21,21 @@ namespace WalletWasabi.WabiSabi.Client;
 
 public class CoinJoinClient
 {
-	private const int MaxInputsRegistrableByWallet = 10; // how many
 	private volatile bool _inCriticalCoinJoinState;
 
 	/// <param name="minAnonScoreTarget">Coins those have reached anonymity target, but still can be mixed if desired.</param>
-	/// <param name="consolidationMode">If true, then aggressively try to consolidate as many coins as it can.</param>
 	public CoinJoinClient(
 		IWasabiHttpClientFactory httpClientFactory,
 		IKeyChain keyChain,
 		IDestinationProvider destinationProvider,
 		RoundStateUpdater roundStatusUpdater,
-		int minAnonScoreTarget = int.MaxValue,
-		bool consolidationMode = false)
+		int minAnonScoreTarget = int.MaxValue)
```

Comment:
Yes. You're right. The owner o that instance should dispose it and i don't know at this point which should be. In case the CJC has to be the owner we have to make it disposable and make everything disposable. I think the CoinJoinManager could be disposable and the owner of the random generator.

We need to fix this by sure but let is out of the scope of this PR,